### PR TITLE
Upgrade Docusaurus to V3 and fixes

### PR DESCRIPTION
**Pull request by maxwolfs**
Upgrade and relevant fixes to the new md rules coming with it. This is only fixing the markdown errors with the markdown files within this repository. There are still 14 compilation that prevent from building. Those have to be fixed in other repositories. See parent issue #113

### CODE REVIEW

File: community/tools/github/dco-and-licenses.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ line to commit messages.
 ```bash
 My fancy commit message
 
-Signed-off-by: Christian Berendt <berendt@betacloud-solutions.de>
+Signed-off-by: Christian Berendt <$berendt@betacloud-solutions.de>
```
Comment (mbuechse) [inline community/tools/github/dco-and-licenses.md:47]: I can live fine with most changes, but this seems wrong.

File: docs/glossary.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ glossary is not intended to evaluate or standardize specific terms.
 
 ## Definition of a Region
 
-An OpenStack/SCS region consists of at least one or more Availability Zones that share a Control Plane with their services. As a result, they share one API. Also a Control Plane can share one CEPH cluster over different fire departments or each Availbility Zone can have its own CEPH cluster. Within the region, any Layer 2 networks are available to the user. Availbility Zones which build a region are connected by redundant low-latency (<2ms (guess!!) ) high bandwidth (10s of Gbps) connections.
+An OpenStack/SCS region consists of at least one or more Availability Zones that share a Control Plane with their services. As a result, they share one API. Also a Control Plane can share one CEPH cluster over different fire departments or each Availbility Zone can have its own CEPH cluster. Within the region, any Layer 2 networks are available to the user. Availbility Zones which build a region are connected by redundant low-latency < 2ms high bandwidth (10s of Gbps) connections.
```
Comment (mbuechse) [inline docs/glossary.md:9]: I think the parentheses should be added back in, to be consistent with the bandwidth. Speaking of consistency though:

```suggestion
An OpenStack/SCS region consists of at least one or more Availability Zones that share a Control Plane with their services. As a result, they share one API. Also a Control Plane can share one CEPH cluster over different fire departments or each Availbility Zone can have its own CEPH cluster. Within the region, any Layer 2 networks are available to the user. Availbility Zones which build a region are connected by redundant low-latency (< 2 ms) high-bandwidth (10s of Gbps) connections.
```

File: community/tools/github/dco-and-licenses.md (hunk 1 above)
Comment (mbuechse) [reply community/tools/github/dco-and-licenses.md]: Screenshot from 2024-04-02 17-28-28
No, this is a code segment, so the escape character will not be interpreted. Why did you put it in there?

**Review by mbuechse — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -37,33 +37,35 @@
     "format": "prettier --log-level warn --write \"./**/*.{jsx,js}\""
   },
   "dependencies": {
-    "@docusaurus/core": "^2.4.3",
-    "@docusaurus/plugin-client-redirects": "^2.4.3",
-    "@docusaurus/preset-classic": "^2.4.3",
-    "@docusaurus/theme-mermaid": "^2.4.3",
-    "@easyops-cn/docusaurus-search-local": "^0.33.6",
+    "@docusaurus/core": "^3.1.1",
```
Comment (berendt) [inline package.json:40]: Docusaurus 3.2.0 is out now. I think it makes sense to change this before the merge of this PR.

**Pull request merged**